performance-tests/custom-typings/k6-httpx/index.d.ts

```diff
@@ -0,0 +1,16 @@
+import { Response } from "k6/http";
+declare module "https://jslib.k6.io/httpx/0.1.0/index.js" {
+
+    export class Httpx {
+        constructor(options: Options);
+        get(...args: any[]): Response;
+        post(...args: any[]): Response;
+        put(...args: any[]): Response;
```

Comment:
> ...args: any[]

These aren't type definitions... All you're doing is turning off typescript.

Reply:
I'm not being able to use this custom typing. Moreover, I'm not sure how to go about this. Is there a way to import types from [`k6/http`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/k6/http.d.ts)?

Reply:
> I'm not being able to use this custom typing.

So you thought that ignoring it is the way to go...? 😑 

I checked it, and it seems like you broke it when you started to define `Response` as the response type. I don't know exactly why this is happening, but I suppose it's because you're importing the whole class (=> the .js file) in a type mappings (.d.ts) file.

> Is there a way to import types from [`k6/http`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/k6/http.d.ts)?

I don't know. I'm afraid you're gonna have to google it.

Reply:
I had to go with manually copying the methods, but it seems to be working now. Also, it seems these custom typings cannot have any kind of imports.